src/Pages/Form/Components/PatientInformationSection.tsx

```diff
@@ -0,0 +1,265 @@
+      borderRadius: "20px",
+    },
+  },
+}))(ToggleButtonGroup);
+
+const OnsetStyledToggleButtonGroup = withStyles((theme) => ({
+  grouped: {
+    textTransform: "none",
+    margin: theme.spacing(3),
+    boxShadow: "0px 2px 4px rgba(0, 0, 0, 0.1)",
+    width: "220px",
+    height: "80px",
+    backgroundColor: "#FFF",
+    color: "#000",
+    border: "none",
+    "&:not(:first-child)": {
+      borderRadius: "20px",
+    },
+    "&:first-child": {
+      borderRadius: "20px",
+    },
+  },
+}))(ToggleButtonGroup);
+
+const PatientInformationSection: React.FC = () => {
+  const classes = useStyle();
+
+  //* Gender Button
+  const [gender, setGender] = React.useState<string | null>("left");
+  const handleGender = (
+    event: React.MouseEvent<HTMLElement>,
+    newGender: string | null
+  ) => {
+    setGender(newGender);
+  };
+
+  //* Onset Button
+  const [onset, setOnset] = React.useState<string | null>("left");
+  const handleOnset = (
+    event: React.MouseEvent<HTMLElement>,
+    newOnset: string | null
+  ) => {
+    setOnset(newOnset);
+  };
+
+  //* Date and Time Picker
+  const [selectedDate, setSelectedDate] = React.useState<Date | null>(
+    new Date()
+  );
+  const handleDateChange = (date: Date | null) => {
+    setSelectedDate(date);
+  };
+
+  return (
+    <Box className={classes.root}>
+      <ThemeProvider theme={theme}>
+        {/* Patient - Text Field */}
+        <Grid container spacing={6}>
+          <Grid item xs={4}>
+            <TextField label="Enter patient ID" className={classes.textField} />
+          </Grid>
+          <Grid item xs={4}>
+            <TextField label="Enter age" className={classes.textField} />
+          </Grid>
+        </Grid>
+        <Grid container spacing={6}>
+          <Grid item xs={4}>
+            <TextField label="Enter first name" className={classes.textField} />
+          </Grid>
+          <Grid item xs={4}>
+            <TextField label="Enter last name" className={classes.textField} />
+          </Grid>
+        </Grid>
+        {/* Gender - Button */}
+        <Box className={classes.textTitle}>
+          <Typography variant="h4">Gender</Typography>
+        </Box>
+        <Grid container spacing={3}>
+          <GenderStyledToggleButtonGroup
+            value={gender}
+            exclusive
+            onChange={handleGender}
+          >
+            <ToggleButton value="male" className={classes.button}>
+              <Box display="flex">
+                <img src={MaleIcon} alt="male icon" />
+              </Box>
+            </ToggleButton>
+            <ToggleButton value="female" className={classes.button}>
+              <Box display="flex">
+                <img src={FemaleIcon} alt="female icon" />
+              </Box>
+            </ToggleButton>
+          </GenderStyledToggleButtonGroup>
+        </Grid>
+        <Box display="flex">
+          <Box className={classes.boxMale}>
+            <Typography variant="body1">Male</Typography>
```

Comment:
As I know, Did you try to set label under the button? If yes, I think it should be implemented in same box of button because if the button is changed position the label should be moved automatically.

Reply:
If you want to set marginTop of Text, you can set to Typography directly.